src/components/CommentList/CommentList.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Comment } from '../Comment';
+
+export const CommentList = ({ comment }) => (
+  <ul>
+    {comment.map(comments => (
```

Comment:
Тарас, расстраиваешь
comment - единственное число
comments - множественное

перебрать можно массив комментов, в колбек приходит один коммент
```suggestion
    {comments.map(comment => (
```